src/storage_module_memory.cpp

```diff
@@ -33,16 +33,22 @@ void StorageModuleMemory::RQRemoveFront(std::shared_ptr<DataEntry> request) {
   }
 }
 
-int StorageModuleMemory::RQCount() {
-  int size = request_queue.size();
+long long StorageModuleMemory::RQCount() {
+  long long size = request_queue.size();
   _logger->log(LogLevel::DEBUG, "[Countly][StorageModuleMemory] RQCount size = " + size);
   return size;
 }
 
 void StorageModuleMemory::RQInsertAtEnd(const std::string &request) {
   _logger->log(LogLevel::DEBUG, "[Countly][StorageModuleMemory] RQInsertAtEnd request = " + request);
   if (request != "") {
-    _lastUsedId += 1;
+    if (request_queue.empty()) {
+      // Reset '_lastUsedId' to 1 if request queue is empty.
```

Comment:
this is a bad comment. you are not explaining the "why" but just stating the obvious